features/step_definitions/auction_status_steps.rb

```diff
@@ -53,8 +53,10 @@
   )
 end
 
-Then(/^I should see the time I placed my bid$/) do
+Then(/^I should see a status confirmation for my sealed bid$/) do
```

Comment:
I wonder if there is a clearer name that "status confirmation" -- feels a bit vague to me